internal/pkg/cli/svc_logs.go

```diff
@@ -187,22 +192,22 @@ func (o *svcLogsOpts) askApp() error {
 }
 
 func (o *svcLogsOpts) askSvcEnvName() error {
-	deployedService, err := o.sel.DeployedService(svcLogNamePrompt, svcLogNameHelpPrompt, o.appName, selector.WithEnv(o.envName), selector.WithSvc(o.svcName))
+	deployedService, err := o.sel.DeployedService(svcLogNamePrompt, svcLogNameHelpPrompt, o.appName, selector.WithEnv(o.envName), selector.WithSvc(o.name))
 	if err != nil {
 		return fmt.Errorf("select deployed services for application %s: %w", o.appName, err)
 	}
-	o.svcName = deployedService.Svc
+	o.name = deployedService.Svc
 	o.envName = deployedService.Env
 	return nil
 }
 
-func (o *svcLogsOpts) parseSince() *int64 {
-	sinceSec := int64(o.since.Round(time.Second).Seconds())
+func (o *wkldLogOpts) parseSince(since time.Duration) *int64 {
```

Comment:
Looks like `parseSince` no longer needs the receiver -
```suggestion
func parseSince(since time.Duration) *int64 {
```